scripts/random_sleep/random_sleep.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+TIMEOUT=$(( $RANDOM % 30 ))
+BASE_TIME=$(( ($RANDOM % 59) + 1 ))
+MULTIPLIER=$(( ($RANDOM % 9) + 1 ))
```

Comment:
What if we replace `BASE_TIME` and `MULTIPLIER` with a clearer statement in the above? Both cases seem to be very similar mathematically but we introduce one variable instead of two different (we anyway need their product).
```suggestion
EXEC_TIME=$(( $RANDOM % (60*10) + 1 ))
```

If we miss something in the current implementation then we would suggest replacing some numbers as follows: 
```suggestion
BASE_TIME=$(( ($RANDOM % 60) + 1 ))
MULTIPLIER=$(( ($RANDOM % 10) + 1 ))
```
This extends ranges for variables: 
|var|current range|suggested range|
|---|---|---|
|`BASE_TIME`|1..59|1..60|
|`MULTIPLIER`|1..9|1..10|

Credit for discussion @solid-vovabeloded

Reply:
Actually the output from EXEC_TIME and BASE * MULTIPLIER methods varies a bit. BASE * MULTIPLIER way is created on purpose, it produces mostly short builds and long ones happen rather rarely. EXEC_TIME approach is purely random.  [1000 hits charts comparison](https://imgur.com/KstMFej)

However it is fine for me, I pushed updated script version.

Reply:
Oh, okay, sorry - now I see what we missed 🤦 
Nevertheless, thank you for the feedback!